seed/index.test.js

```diff
@@ -0,0 +1,13 @@
+const seed = require('./index.js');
+
+
+
+test('generates 100 house listings', () => {
+    const houses = seed();
+    expect(houses.length).toBe(100);
+})
+
+test('generates listing ids as zero padded strings', () => {
+    const house = seed()[1];
+    expect(house['Listing_id']).toBe('001');
```

Comment:
Maybe you also want to check the 100th or 101st entry?  That way you've tested both extremes of your data-set - and if you set it up to dynamically check for the last entry your test will continue to verify any additional db entries (which may or may not come up,  just trying to think critically about testing while avoiding making them brittle.)

Reply:
Thanks, I hadn't put much thought into the testing side yet, that is a good idea